tests/components/jewish_calendar/test_sensor.py

```diff
@@ -568,3 +568,37 @@
         await hass.async_block_till_done()
 
     assert hass.states.get("sensor.test_day_of_the_omer").state == result
+
+
+DAFYOMI_PARAMS = [
+    (dt(2014, 4, 28, 0), "Beitzah 29"),
+    (dt(2020, 1, 4, 0), "Niddah 73"),
+    (dt(2020, 1, 5, 0), "Berachos 2"),
+    (dt(2020, 3, 7, 0), "Berachos 64"),
+    (dt(2020, 3, 8, 0), "Shabbos 2"),
+]
+DAFYOMI_TEST_IDS = [
+    "randomly_picked_date",
+    "end_of_cycle13",
+    "start_of_cycle14",
+    "cycle14_end_of_berachos",
+    "cycle14_start_of_shabbos",
+]
+
+
+@pytest.mark.parametrize(["test_time", "result"], DAFYOMI_PARAMS, ids=DAFYOMI_TEST_IDS)
+async def test_dafyomi_sensor(hass, test_time, result):
+    """Test Daf Yomi sensor output."""
+    test_time = hass.config.time_zone.localize(test_time)
+
+    with alter_time(test_time):
+        assert await async_setup_component(
+            hass, jewish_calendar.DOMAIN, {"jewish_calendar": {"name": "test"}}
+        )
+        await hass.async_block_till_done()
+
+        future = dt_util.utcnow() + timedelta(seconds=30)
+        async_fire_time_changed(hass, future)
+        await hass.async_block_till_done()
+
+    assert hass.states.get("sensor.test_daf_yomi").state == result
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5011603)